src/analyze/report.ts

```diff
@@ -21,7 +22,7 @@ export interface ReportResult {
   dependencies: DependencyStats;
 }
 
-const plugins: ReportPlugin[] = [runAttw, runPublint, runReplacements];
+let plugins: ReportPlugin[] = [runAttw, runPublint, runReplacements, runKnip];
```

Comment:
this should stay as it was (`const`) and be a `defaultPlugins`

we should have a local `plugins` variable inside `report`